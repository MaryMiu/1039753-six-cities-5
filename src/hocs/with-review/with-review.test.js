import React from 'react';
import renderer from 'react-test-renderer';
import withReview from './with-review';

const MockComponent = () => {
  return <div />;
};

const MockComponentWrapped = withReview(MockComponent);

it(`withReview is render correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      rating={5}
      comment={`Тест`}
      isDisabled={true}
      onChangeForm={() => {}}
      onChangeDisabledState={() => {}}
      resetForm={() => {}}
    />
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
