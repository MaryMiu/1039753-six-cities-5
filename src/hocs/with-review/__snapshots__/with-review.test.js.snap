// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withReview is render correctly 1`] = `<div />`;
